x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/ClassificationIT.java

```diff
@@ -377,4 +343,26 @@ private static void indexData(String sourceIndex, int numTrainingRows, int numNo
         // Assert that the top classes are listed in the order of decreasing probabilities.
         assertThat(Ordering.natural().reverse().isOrdered(classProbabilities), is(true));
     }
+
+    private void assertEvaluation(String dependentVariable, List<String> dependentVariableValues, String predictedClassField) {
+        EvaluateDataFrameAction.Response evaluateDataFrameResponse =
+            evaluateDataFrame(
+                destIndex,
+                new org.elasticsearch.xpack.core.ml.dataframe.evaluation.classification.Classification(
```

Comment:
java needs import alias :)

Reply:
Fully agree...
Another option would be to rename the class to `ClassificationEvaluation` (similarly with `Regression`) but this is outside the scope of this PR